### PR TITLE
Suite form

### PR DESCRIPTION
**Pull request by MitchellWT**
This MR does the following:
- Hooks up suite form from the front end to the backend.
- Made backend response correctly to frontend form.

NOTE: This MR will need to merged after the react-electron branch.

### CODE REVIEW

File: gui/src/components/SuiteForm.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    console.log(`Message to send: ${msg}`);
+    console.log(`Socket state: ${socket.readyState}`);
+    if (socket.readyState === 1) socket.send(msg);
+  }
+
+  return (
+    <Form ref={formRef} className="w-25 py-2 px-4 bg-secondary text-white d-flex flex-column justify-content-between">
+      <Form.Group>
+        <Form.Label>URL</Form.Label>
+        <Form.Control name="url" type="text" placeholder="google.com"/>
```
Comment (Oblivion-21) [inline gui/src/components/SuiteForm.js:39]: You need to pass the protocol otherwise the request will fail, update to https://google.com

**Review by Oblivion-21 — CHANGES_REQUESTED** [review]
No real issues (bar the one small comment) but I don't know JS so maybe wait on Daniels review

File: gui/src/components/SuiteForm.js (hunk 1 above)
Comment (MitchellWT) [reply gui/src/components/SuiteForm.js:39]: yes, I'm not sure If the tests should prepend that or not, what do you think?

Comment (Oblivion-21) [reply gui/src/components/SuiteForm.js:39]: The way I see if we have 2 choices
- Make users append `https://` to all the functions
- Added that into Daniels web request function al la f"https://{url}" 

The latter might be a better way but maybe we should discuss it tonight

Comment (MitchellWT) [reply gui/src/components/SuiteForm.js:39]: Agreed

**Pull request merged**